tests/qa_automation/acceptance.pm

```diff
@@ -65,8 +65,25 @@ sub run() {
     # QA DB upload happens for each module
     # output the failed tests to serial console
     type_string "export PS1=#\n";
+    # qa_testset_automation deleted oldlog folders, so we add it back to adopt it
+    assert_script_run "mkdir /var/log/qa/oldlogs; cd /var/log/qa/oldlogs; for i in /var/log/qaset/log/*.bz2; do tar -xjvf \$i; done";
```

Comment:
i is normally used for integers, so it is a bit misleading here. better f or file